.golangci.yml

```diff
@@ -0,0 +1,18 @@
+linters-settings:
+  custom:
+    nomaprange:
+      path: .nomaprange.so
+      description: Checks for range over maps
+      original-url: github.com/Kubuxu/go-no-map-range
```

Comment:
can you help me find source?  If I go to https://github.com/Kubuxu/go-no-map-range I get "no branches"

Reply:
Sorry, I've forgot to push a branch (pushed just a tag for go.mod).